### DISCUSSION TITLE
complete replacement for raspberry pi ?

### DISCUSSION THREAD

**Discussion by HiIamChaitanya** [feelfreelinux/octo4a#428] [General]
I wanted to use my Android as a complete replacement for raspberry pi and use it for longer prints does it affect any print and hardware in the long run?

thanks

<printer> - core xy style, Marlin 2.0.9 LTS, MKS TinyBee Board, 12V System with heated bed.

**Comment by kdobrev** [comment]
It was working for me without any problems, but just started having no response from the printer. I tested with a normal octoprint and it did work, so maybe the andrioid base has something to do with it

**Reply by HiIamChaitanya** [reply]
oh, thanks.
so it's just a temporary replacement.

**Reply by gedascher** [reply]
I've been using it with my Ender 3 Pro for quite a while now and it is pretty solid. Every once in a while, I have to restart the app on the phone if I haven't used it in a while. Other than that, I'm super happy with it.  I do NOT have a webcam currently. I tried using another Android phone with an IP Webcam app, but I have to restart THAT app nearly every time I go to print something.
